### ISSUE TITLE
Can't perform extra construction step on Reactor Type X / Reactor Type V

### ISSUE THREAD

**Issue by dmfrodrigues** [Dubwise56/Rimatomics#26]
TL;DR: Can build Reactor Type X but my pawns won't complete the extra construction step.

OS: Windows
Rimworld version: v1.3.3072
Rimatomics version: v1.7.2439

I've already researched Reactor Type A and I saw my pawns completing the extra construction step on the Reactor Type A (which is reactor Alpha). That reactor is working just fine.

I'm mid-way into researching Reactor Types X and V, and I'm on the research step where I'm supposed to build a Type X or Type V reactor.

![image](https://user-images.githubusercontent.com/43684166/132259786-b0772be4-13ce-41e4-8bb8-bf968c36ecad.png)

I've managed to build it and changed its ID to Beta, but I can't complete the extra construction step, so it just stays forever looking like that extra construction step was not completed (which it wasn't, as I didn't see any pawns completing that extra step).

![image](https://user-images.githubusercontent.com/43684166/132259889-c328c9dc-f55e-49f8-a062-622af845e26e.png)

When I go to the reactor console I get this:

![image](https://user-images.githubusercontent.com/43684166/132259941-115fef3b-a917-4bb0-a29b-217e9f1cf411.png)

So it says it is not built, but I can still designate slots to be filled. They are indeed filled with rods (somehow), but when I start the reactor and (try to) pull out the rods up to 100% nothing happens (i.e., the reactor does not work).

Curiously, when I build a new Type A reactor, I don't have to go through the extra construction step. I'm not sure if that was as expected, but I surely didn't expect it.

Using the most recent version from master branch (commit e7f1ca6), I get a similar behavior, except when I open the console, instead of being able to designate slots and raise/lower rods, there is nothing (which I suppose is the expected behavior); nonetheless none of my pawns are completing the extra construction step.

I'm using versions from this repo only, not from Steam.

I'm using some other mods, tho I believe they should not be interfering.

**Comment by dmfrodrigues** [comment]
Hi again! So I've entered god mode and reset all Rimatomics research. Also disabled a few mods that do not have anything to do (except maybe RocketMan, I'll get to that later).

I've used god mode to research everything instantly until reaching the same point I was at before. And guess what now I can complete the extra construction step. So I believe it had something to do with something not being remembered so Rimatomics was struggling to allow me to complete that extra step.

Now back to RocketMan: I added Rimatomics to an existing savefile, and some time later added RocketMan. I am not sure as I do not have any evidence, but I believe there is a slight chance that RocketMan messed with something that Rimatomics needs.
